pkg/build/gcb/executor.go

```diff
@@ -259,8 +261,14 @@ func (e *Executor) doBuild(ctx context.Context, handle *gcbHandle, cloudBuild *c
 	return waitMetadata.Build, nil
 }
 
+func buildTag(desc, val string) string {
+	combined := fmt.Sprintf("%s-%s", desc, val)
+	combined = regexp.MustCompile(`[^\w.-]`).ReplaceAllString(combined, "")
```

Comment:
nit: we could move the regex to a top-level constant and only ReplaceAll on the `val` to avoid re-assigning combined.